espnet2/asr/encoder/beats_encoder.py

```diff
@@ -267,6 +267,7 @@ def __init__(
             self.cross_embed_positions = BartLearnedPositionalEmbedding(
                 max_positions, learned_pos_dim
             )
+        self.min_input_length_at_16khz = 3200
```

Comment:
where does this number (3200) come from?

Reply:
When input is very small the beats code gives me errors (window sizes become larger than input length). So I just tried to check at what values the code works and set that. This is not a good way, maybe I should just remove this and filter small samples(?)

Reply:
It's OK for now, but can you add some comments here, and also define the variable so that we might change it later?

In the future, we should handle this with some option argument.
Please add your TODO and also add the FIXME flag